api-lib/circleAdmin.ts

```diff
@@ -37,9 +42,9 @@ const middleware =
       }
       throw new UnauthorizedError('User not circle admin', err);
     }
-    // the admin role is validated early by zod
```

Comment:
This is/was a key comment

Reply:
restored in the handler for the admin case